mikeio1d/result_network/result_reach.py

```diff
@@ -101,11 +131,15 @@ def set_gridpoint(self, reach, gridpoint):
         """
         current_reach_result_gridpoints = self.current_reach_result_gridpoints
 
-        result_gridpoint = ResultGridPoint(reach, gridpoint, reach.DataItems, self, self.res1d)
```

Comment:
Is black configured to reformat for 80 characters? I thought we were using 120 character convention. For me this break is redundant.